glean-core/csharp/Glean/Metrics/LabeledMetricType.cs

```diff
@@ -56,6 +56,9 @@ public sealed class LabeledMetricType<T> where T : ILabeledSubmetricInterface
             Func<string, string, string[], int, int, bool, string[], int, UInt64> metricTypeInstantiator;
             switch (submetric)
             {
+                case CounterMetricType _:
```

Comment:
Nit: Are we matching the order in the other bindings?  I'm not opposed to putting this here, but maybe some order like alphabetical might be a nice way to sort the cases?

Reply:
I was following the other bindings, but you're right and it's better to sort alphabetically.